command/plugin.go

```diff
@@ -24,7 +24,6 @@ import (
 	azuredtlbuilder "github.com/hashicorp/packer/builder/azure/dtl"
```

Comment:
We can stop making this file autogenerated from now on. It will be easier to add imports ourselves. 
We can keep the script though as an option to run it manually and write the structure of the file.
Otherwise, it will always overwrite the manual changes.